store/gcworker/gc_worker.go

```diff
@@ -1949,7 +1949,7 @@ func (w *GCWorker) doGCLabelRules(dr util.DelRangeTask) (err error) {
 			return
 		}
 
-		patch := label.NewRulePatch(nil, ruleIDs)
+		patch := label.NewRulePatch([]*label.Rule{}, ruleIDs)
 		err = infosync.UpdateLabelRules(context.TODO(), patch)
```

Comment:
Because this PR force pushed too much, I don't get why do we need non-nil empty slice? What is the reason behind?

Reply:
I think it's more readable?

Reply:
OK... I prefer nil though.